core/store/src/trie/mod.rs

```diff
@@ -1459,7 +1459,13 @@ mod tests {
 
         assert_eq!(trie3.get(b"dog"), Ok(Some(b"puppy".to_vec())));
         assert_eq!(trie3.get(b"horse"), Ok(Some(b"stallion".to_vec())));
-        assert_eq!(trie3.get(b"doge"), Err(StorageError::MissingTrieValue));
+
+        match trie3.get(b"doge") {
```

Comment:
Check out assert_matches! macro, there are some usages of it in the codebase, it be handy here.